Doc/tutorial/errors.rst

```diff
@@ -108,8 +108,9 @@ The :keyword:`try` statement works as follows.
 
 * If an exception occurs which does not match the exception named in the *except
   clause*, it is passed on to outer :keyword:`try` statements; if no handler is
-  found, it is an *unhandled exception* and execution stops with a message as
-  shown above.
+  found, it is an *unhandled exception* and execution stops with error message.
+  The error message will provide information about the *unhandled exception*,
+  as described in the :ref:`tut-exceptions` section.
```

Comment:
The report says:

> If I understand correctly, I believe it will be clearer if "and execution stops with a message as shown above" is shortened to simply "and execution stops".

Let's do something halfway:

```suggestion
  found, it is an *unhandled exception* and execution stops with an error message.
```